tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -35,8 +35,10 @@
  * exception is thrown during a mutating change, the change is aborted. It is safe to continue to
  * use a {@code QueueFile} instance after an exception.
  *
- * <p>All operations are synchronized. In a traditional queue, the remove operation returns an
- * element. In this queue, {@link #peek} and {@link #remove} are used in conjunction. Use
+ * <p><strong>Note that this implementation is not synchronized.</strong>
```

Comment:
@f2prateek it's at the class level. Are you confused by the diff showing a jump from line 44 to 300 where `add()` javadoc starts?